PEPRest/pom.xml

```diff
@@ -14,15 +14,14 @@
 	<version>2.0-SNAPSHOT</version>
 
 	<properties>
-		<jclouds.version>1.9.1</jclouds.version>
-		<springfox.version>2.2.2</springfox.version>
-		<swagger.version>1.5.3</swagger.version>
+		<springfox.version>2.5.0</springfox.version>
```

Comment:
can't update to the lastest 2.9.2
updating this strangely breaks configuration properties with a prefix
UCS-114 task